nautobot/utilities/tables.py

```diff
@@ -382,3 +402,63 @@ def render(self, record, bound_column, value):
             template = escape(value)
 
         return mark_safe(template)
+
+
+class RelationshipColumn(tables.Column):
+    """
+    Display relationship association instances in the appropriate format.
+    """
+
+    def __init__(self, relationship, side, *args, **kwargs):
+        self.relationship = relationship
+        self.side = RelationshipSideChoices.OPPOSITE[side]
```

Comment:
Feels weird to me that `self.side` is the opposite of the `side` passed in to `__init__`. Is this the most intuitive way to do things?